token-lending/program/tests/fixtures/README.md

```diff
@@ -4,10 +4,10 @@
 
 ```
 $ solana config set --url https://api.mainnet-beta.solana.com
-$ solana account 7xMDbYTCqQEcK2aM9LbetGtNFJpzKdfXzLL5juaLh4GJ --output-file sol_usdc_dex_market.bin
-$ solana account 4VndUfHkmh6RWTQbXSVjY3wbSfqGjoPbuPHMoatV272H --output-file sol_usdc_dex_market_bids.bin
-$ solana account 6LTxKpMyGnbHM5rRx7f3eZHF9q3gnUBV5ucXF9LvrB3M --output-file sol_usdc_dex_market_asks.bin
-$ solana account CDdR97S8y96v3To93aKvi3nCnjUrbuVSuumw8FLvbVeg --output-file srm_usdc_dex_market.bin
-$ solana account DkxpXtF1EyjHomQcEhH54498gdhUN3t1sZCjReFNYZZn --output-file srm_usdc_dex_market_bids.bin
-$ solana account DRqgRZqfdD6PLHKSU7ydyVXWMUpvkqhzLZ1JSKn1iB1K --output-file srm_usdc_dex_market_asks.bin
+$ solana account 9wFFyRfZBsuAha4YcuxcXLKwMxJR43S7fPfQLusDBzvT --output-file sol_usdc_dex_market.bin
+$ solana account 14ivtgssEBoBjuZJtSAPKYgpUK7DmnSwuPMqJoVTSgKJ --output-file sol_usdc_dex_market_bids.bin
+$ solana account CEQdAFKdycHugujQg9k2wbmxjcpdYZyVLfV9WerTnafJ --output-file sol_usdc_dex_market_asks.bin
+$ solana account ByRys5tuUWDgL73G8JBAEfkdFf8JWBzPBDHsBVQ5vbQA --output-file srm_usdc_dex_market.bin
+$ solana account AuL9JzRJ55MdqzubK4EutJgAumtkuFcRVuPUvTX39pN8 --output-file srm_usdc_dex_market_bids.bin
+$ solana account 8Lx9U9wdE3afdqih1mCAXy3unJDfzSaXFqAvoLMjhwoD --output-file srm_usdc_dex_market_asks.bin
```

Comment:
Where/how do you find these addresses?

Reply:
Found the markets on https://dex.projectserum.com and then searched for recent transactions which used bids/asks accounts.